joomla/joomla-cms
# [4.3]No search result found msg in media new

**Newfeature**

Alternate pr for pr #39819 

Summary of Changes
ADD a messge that will display when no result found on searching in media manager

Testing Instructions
Search in media manager

Actual result BEFORE applying this Pull Request
No message shown if no result match .

Expected result AFTER applying this Pull Request
A message will shown if result not match with search
